statsmodels/regression/process_regression.py

```diff
@@ -66,8 +66,7 @@ class ProcessCovariance(object):
     ----------
     Paciorek, C. J. and Schervish, M. J. (2006). Spatial modeling using
     a new class of nonstationary covariance functions. Environmetrics,
-    17:483–506.
-    https://papers.nips.cc/paper/2350-nonstationary-covariance-functions-for-gaussian-process-regression.pdf
+    17:483–506. https://tinyurl.com/ycpcklsb
```

Comment:
I don't like url shortening sites. They look too much like spam and don't indicate which site is accessed

Reply:
I agree, but I was getting the lint errors below when I included the original URL.  I'll revert and see what we can do.

```
Linting all files with limited rules
Linting known clean files with strict rules
fatal: --unshallow on a complete repository does not make sense
Linting newly added files with strict rules
statsmodels/regression/process_regression.py:70:-1235: W605 invalid escape sequence '\e'
statsmodels/regression/process_regression.py:70:-993: W605 invalid escape sequence '\c'
statsmodels/regression/process_regression.py:70:-982: W605 invalid escape sequence '\c'
statsmodels/regression/process_regression.py:70:-952: W605 invalid escape sequence '\s'
statsmodels/regression/process_regression.py:70:-920: W605 invalid escape sequence '\c'
statsmodels/regression/process_regression.py:70:-886: W605 invalid escape sequence '\s'
New files failed linting.
travis_time:end:03de94f9:start=1546284649417345111,finish=1546284716640134865,duration=67222789754
[0K[31;1mThe command "./lint.sh" exited with 1.[0m
```